### DISCUSSION TITLE
Thank you for this guide

### DISCUSSION THREAD

**Discussion by thiagoveloso** [jimpalompa/OpenCore-Gigabyte-Z370n-Wifi#7] [General]
@jimpalompa thank you for this excellent guide. It is by far the best one I have seen for the Z370N WiFi. I am sure my system is running faster and smoother than ever.

There were only a few points that I was left wondering how to do:

1. replacing your bluetooth driver with one that is compatible with my Broadcom card; and
2. adding a more beautiful boot menu to Opencore

However, it took me only like half an hour to figure out those points by myself.

Apart from that, overall excellent and I am very grateful!

**Comment by jimpalompa** [comment]
Thanks for kudos 💛

Since this configuration is only for my build, with certain hardware, I cannot write a guide for other hardware. Though, Dortania has excellent guides for your issues!

1. Read the Wireless Buyers Guide here and find you broadcom model and setup:
https://dortania.github.io/Wireless-Buyers-Guide/

2. To add GUI boot menu, you need to follow this guide. I prefer a non-bloated EFI, you'll probably understand what I mean, when you read the guide ; )
https://dortania.github.io/OpenCore-Post-Install/cosmetic/gui.html

Have fun, cheers!
